operations-guide/runtime-tuning/tuning-event-processing.md

```diff
@@ -75,20 +75,20 @@ CompletableFuture<Boolean> futureResult = trackingProcessor.mergeSegment(segment
 > Such a set up is a regular scenario to require segment tuning.
 >
 > Note though that especially in such a set up you would need to delegate said split or merge to the correct instance.
-> With "correct instance" is meant the instance owning the segment you want to split and merge.
+> Where "correct instance" is means the instance owning the segment you want to split and merge.
```

Comment:
It might be me, as I am not a native English speaker, but this adjustment doesn't seem correct to me. Would you mind elaborating why you've changed it to this?

Reply:
The adjustment isn't quite correct either. I should have removed the `is` as well.

The pattern `where ___ means` is the way a native speaker would convey this.

For example: `The cells in a tick-tack-toe board are represented by a 2-dimensional array. Each cell in the array is a character. Each cell contains an X, O, or an _ (where _ means the cell is empty).`

I think the following would would be more consistent with what seems like the original intent.
 `
Note though that especially in such a setup you would need to delegate said split or merge to the correct instance (where "correct instance" means the instance owning the segment you want to split and merge).
`

However, I think it *could* be even more clear by simply stating:
`
Note that, especially, in such a setup you would need to delegate said split or merge to the correct instance. The "correct instance", in this case, is the instance owning the segment you want to split and merge.
`

Reply:
Liking your last suggestion there @chutch1122! If you would mind making that adjustment in this PR, that would be great. :-)